src/main/java/redis/clients/jedis/CommandObjects.java

```diff
@@ -4168,6 +4172,11 @@ public void setJsonObjectMapper(JsonObjectMapper jsonObjectMapper) {
     this.jsonObjectMapper = jsonObjectMapper;
   }
 
+  public void setDefaultSearchDialect(int dialect) {
```

Comment:
The issue says ```This configuration option should be optional. If it is not specified, the client should fall back to the current behavior.```

Raising this argument isn't that.

Reply:
Firstly, users would be using the one in UnifiedJedis. I guess we're concerned about that and so check my answer in test file.